addons/my-ocular/profile.js

```diff
@@ -1,46 +1,30 @@
 export default async function ({ addon, console, msg }) {
-  const username = document.querySelector("#profile-data > div.box-head > div > h2").innerText?.split("#")[0]; // Fix bug with user-id addon
-  let container = document.querySelector(".location");
+  const username = Scratch.INIT_DATA.PROFILE.model.username;
   let data = await fetchStatus(username);
 
-  addon.settings.addEventListener("change", updateOcular);
-  addon.self.addEventListener("disabled", () => updateOcular(true));
-  addon.self.addEventListener("reenabled", () => updateOcular());
-
   if (typeof data.userStatus !== "string") return;
 
-  // Create status element
-  let statusSpan = document.createElement("span");
-  statusSpan.title = msg("status-hover");
-  statusSpan.innerText = data.userStatus;
-  statusSpan.style.fontStyle = "italic";
-  statusSpan.style.setProperty("display", "inline-block", "important");
-  statusSpan.id = "my-ocular-span";
+  const span = document.createElement("span");
+  span.textContent = data.userStatus;
+  span.className = "sa-ocular-status";
+
+  span.style.fontStyle = "italic";
+  span.style.display = "inline-block !important";
 
-  // Create my-ocular-dot
-  let dot = document.createElement("span");
+  const dot = document.createElement("span");
   dot.title = msg("status-hover");
```

Comment:
The title attribute is in the wrong element now.
Hovering the whole status should let you see it, not just hovering the dot.